docs/examples.md

```diff
@@ -76,6 +76,24 @@ $y$j9T$A0Y3wwVOKP69S.1K/zYGN.$S596l11UGH3XjN...
 
 The `yescrypt` hashing method is recommended for new passwords. For more details on hashing methods, see `man 5 crypt`.
 
+In case inline definitions of SSH keys are undesirable you can use file references instead. This example creates three users (`core`, `jlebon`, `miabbott`) and sets up their respective SSH keys via file references instead of inlining them. This requires using `butane` with the `--files-dir` parameter to allow loading files from disk.
+
+### Handling SSH public keys
+
+As shown in the previous examples you can inline multiple SSH public keys per user directly in the config file. Additionally, you can embed keys from local files at transpile time. This example creates a `core` user and configures its SSH keys from local files. The file paths are relative to the directory specified with the `--files-dir` command-line option, which must be provided.
+
+<!-- butane-config -->
+```yaml
+variant: fcos
+version: 1.5.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys_local:
+        - id_rsa.pub
+        - id_ed25519.pub
+```
+
 For more information, see the Fedora CoreOS documentation on [Authentication][fcos-auth-docs].
```

Comment:
It's a bit ambiguous, but from the Git history it seems this was intended to be part of the password auth section.

Reply:
Technically true, however, as pointed out in the [fcos-docs#511](https://github.com/coreos/fedora-coreos-docs/issues/511) issue and related PR (I should have another look at 🙈) the authentication page would be a good place for additional / more complex examples for SSH public key handling as well.

I'll move it back to the password settings section anyway.